stdlib/src/builtin/simd.mojo

```diff
@@ -576,13 +580,47 @@ struct SIMD[type: DType, size: Int = simdwidthof[type]()](
 
         return String.format_sequence(self)
 
+    @always_inline
+    fn __repr__(self) -> String:
+        """Get the representation of the SIMD value eg. "SIMD[DType.int8, 2](1, 2)".
+        Returns:
+            The representation of the SIMD value.
+
+        """
+        var values: String
+
+        var output = String()
+        var writer = output._unsafe_to_formatter()
+        self.format_to(writer, True)
+
+        @parameter
+        if size > 1:
+            values = output[1:-1]
+        else:
+            values = output^
+        return (
+            "SIMD[" + type.__repr__() + ", " + str(size) + "](" + values + ")"
+        )
+
+    # overload required so SIMD still conforms to Formattable
+    @always_inline
     fn format_to(self, inout writer: Formatter):
         """
         Formats this SIMD value to the provided formatter.
 
         Args:
             writer: The formatter to write to.
         """
+        self.format_to(writer, False)
+
+    fn format_to(self, inout writer: Formatter, use_scientific_notation: Bool):
```

Comment:
Why not use a default value instead of an overload?
```suggestion
    fn format_to(self, inout writer: Formatter, use_scientific_notation: Bool = False):
```

Reply:
Doing so results in a compiler error in `SIMD.__str__()` when it tries to call `String.format_sequence(self)` since I thats not the function signature it expects of a `Formattable` type (I think?)

`invalid call to 'format_sequence': callee with non-empty variadic pack argument expects 0 positional operands, but 1 was specified`

Reply:
Right, thanks! Could you please just mention this in a comment or the docstring?